### PR TITLE
Update dependencies and patch conditional-section

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Update to non-broken version of component-boilerplate, wait for jwt before querying user data on conditional-section to avoid flashing

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: (PUT IT HERE)
 - Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist
- [ ] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/bedrock-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/bedrock-components",
-  "version": "1.1.0",
+  "version": "1.1.1-1",
```
Comment (loganvolkers) [inline packages/bedrock-components/package.json:3]: Should be 1.1.1

File: packages/bedrock-components/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.1] - 2021-09-14
```
Comment (loganvolkers) [inline packages/bedrock-components/CHANGELOG.md:10]:
```suggestion
## [1.1.1] - 2021-09-17
```

**Review by loganvolkers — COMMENTED** [review]
Looks good to me! Needs the changelog and NPM versions to be properly updated, then can be deployed.

File: packages/bedrock-components/package.json (hunk 1 above)
Comment (00salmon) [inline packages/bedrock-components/package.json:3]:
```suggestion
  "version": "1.1.1",
```

**Review by loganvolkers — APPROVED** [review]
Looks good to me

**Pull request merged**